openreview/arr/process/submission_preprocess.py

```diff
@@ -35,6 +42,13 @@ def process(client, edit, invitation):
 
         if (arr_submission_v1 and 'aclweb.org/ACL/ARR' not in arr_submission_v1.invitation) or (arr_submission_v2 and not any('aclweb.org/ACL/ARR' in inv for inv in arr_submission_v2.invitations)):
             raise openreview.OpenReviewException('Provided paper link does not correspond to an ARR submission')
+            
+        # Get the current venue ID to check if it's from the current cycle
+        venue_id = invitation.id.split('/-/')[0]
```

Comment:
It's more reliable to get the domain
```suggestion
        venue_id = invitation.domain
```